### ISSUE TITLE
Алгоритмы: Бинарный поиск

### ISSUE THREAD

**Issue by cergmin** [doka-guide/content#2216]
Не знаю, насколько описание алгоритмов уместно в документации по web, ведь они применяются и в других областях, но можно добавить их в раздел "JavaScript" в отдельный подраздел. Готов этим заняться.

**Comment by nlopin** [comment]
Давайте попробуем. Нужно составить план по алгоритмам и определиться с подходом. Я создал RFC, давайте там обсуждать

**Comment by nlopin** [comment]
@cergmin давай начнём с вводной статьи? #2269

**Comment by cergmin** [comment]
@nlopin, да, хорошо. В ближайшее время допишу статью про Markdown, а потом возьму за вводную для алгоритмов.
